apis/beacon/genesis.yaml

```
@@ -13,9 +13,16 @@
             schema:
               type: object
               properties:
-                genesis_time:
-                  $ref: '../../beacon-node-oapi.yaml#/components/schemas/GenesisTime'
-                genesis_validators_root:
-                  $ref: '../../beacon-node-oapi.yaml#/components/schemas/Root'
+                data:
+                  type: object
+                  properties:
+                    genesis_time:
+                      $ref: '../../beacon-node-oapi.yaml#/components/schemas/GenesisTime'
+                    genesis_validators_root:
+                      $ref: '../../beacon-node-oapi.yaml#/components/schemas/Root'
+                    genesis_fork_version:
+                      $ref: '../../beacon-node-oapi.yaml#/components/schemas/ForkVersion'
+      "404":
+        description: "Chain genesis has not yet occurred"
```

Maybe the following is a bit more accurate because there is a time in which you know the genesis data before genesis actually happens

```suggestion
        description: "Chain genesis info is not yet known"
```